livingatlas/pipelines/src/main/java/org/apache/beam/sdk/io/solr/SolrIO.java

```diff
@@ -693,6 +682,14 @@ private void flushBatch() throws IOException, InterruptedException {
                     exception);
 
               } else {
+
+                LOG.warn("### Re-connecting to SOLR client in SOLRIO");
```

Comment:
@djtfmartin - is this also a SolrIO deviation?

Reply:
yep - ill remove